src/app/desktop/windows/terminal/terminal-states.ts

```diff
@@ -773,6 +785,10 @@ export class DefaultTerminalState extends CommandTerminalState {
     });
   }
 
+  historyClear() {
+    this.protocol = [];
```

Comment:
Better attribute name + define somewhere with an empty array at the top.

Reply:
You mean to give `protocol` a better name?